integtest/src/main/java/org/corfudb/universe/cluster/vm/VmCluster.java

```diff
@@ -0,0 +1,33 @@
+package org.corfudb.universe.cluster.vm;
+
+import com.google.common.collect.ImmutableList;
+import org.corfudb.universe.cluster.Cluster;
+import org.corfudb.universe.cluster.ClusterException;
+import org.corfudb.universe.service.Service;
+
+/**
+ * This implementation provides a Cluster servers each of which a VM machine.
+ * Deploy is going to deploy a list of servers
+ */
+public class VmCluster implements Cluster {
+
+    @Override
+    public Cluster deploy() throws ClusterException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.corfudb.universe.cluster.ClusterException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)